Tools/clinic/clinic.py

```diff
@@ -2164,11 +2169,11 @@ def add_destination(
             self,
             name: str,
             type: str,
-            *args
+            *args: str
```

Comment:
I'm typing confused here; how should I interpret this?

Reply:
It means that each `arg` in the `*args` tuple should be of type `str`

Reply:
So any of these would be okay:

`self.add_destination(foo', 'bar')`
`self.add_destination(foo', 'bar', 'baz')`
`self.add_destination(foo', 'bar', 'baz', 'eggs')`

But this would not:

`self.add_destination(foo', 'bar', 42)`

And neither would this:

`self.add_destination(foo', 'bar', ('foo', 'bar'))`